components/TipOfTheDay/index.tsx

```diff
@@ -1,11 +1,34 @@
+import { useEffect, useState } from "react"
 import styles from "./TipOfTheDay.module.css"
+import { TIPS } from "../../data/tipsOfTheDay"
 
-export const TipOfTheDay = () => {
-	return (
-		<div className={styles.card}>
-			<h2>Short tip title</h2>
-			<p>Longer explanation of tip</p>
-			<a href="#">Name of Link</a>
-		</div>
-	)
+interface ITip {
+	title: string
+	description: string
+	tipLink: string
+}
+
+export const TipOfTheDay: React.FC = () => {
+	const [randomIndex, setRandomIndex] = useState<number>(0)
+
+	useEffect(() => {
+		const TIPS_LENGTH = TIPS.length
+		const randomNumber = Math.floor(Math.random() * TIPS_LENGTH)
+
+		setRandomIndex(randomNumber)
+	})
+
+	const createTipOfTheDay = () => {
+		const tipIndex: ITip = TIPS[randomIndex]
+		return (
+			<>
+				<h2>{tipIndex.title}</h2>
+				<p>{tipIndex.description}</p>
+				<a target="_blank" rel="noreferrer" href={tipIndex.tipLink}>
+					Read More
+				</a>
```

Comment:
If this opens in a new window we need to specify that clearly for the user. Also, `read more` as a link text is very generic. Either it needs an aria label which adds the extra "Read more about ...tip topic" or we do something like make the heading itself into a link.